app/test/shared/versions_test.dart

```diff
@@ -61,41 +61,16 @@ void main() {
 
   group('dartdoc serving', () {
     test('old versions are serving', () {
-      expect(shouldServeDartdoc('0.16.0', '0.1.6', '0.0.0'), isTrue);
+      expect(shouldServeDartdoc(null), isTrue);
+      expect(shouldServeDartdoc('2017.1.1'), isTrue);
     });
 
-    test('max versions are serving', () {
-      expect(
-          shouldServeDartdoc(
-            dartdocVersion,
-            flutterVersion,
-            customizationVersion,
-          ),
-          isTrue);
+    test('current version is serving', () {
+      expect(shouldServeDartdoc(dartdocVersion), isTrue);
```

Comment:
This should be the current `runtimeVersion`, right?

Reply:
yes, this is the current

Reply:
It's the current dartdoc version ('0.17.0'), but we're comparing it against the current `_dartdocServingRuntime` (2018.3.8), if I'm reading the code right.

Reply:
ah, yeah, my mistake, fixing